scalactic-test/src/test/scala/org/scalactic/TypeCheckedMapEqualityConstraintsSpec.scala

```diff
@@ -33,7 +33,7 @@ class TypeCheckedMapEqualityConstraintsSpec extends FunSpec with NonImplicitAsse
   val sub1: Sub = new Sub(1)
   val super2: Super = new Super(2)
   val sub2: Sub = new Sub(2)
-  val nullSuper: Super = null
+  val nullSuper: Super | Null = null
```

Comment:
**Nullable variable**
Variables that really can be null and should therefore have a `| Null` type.